docs/statistics.md

```diff
@@ -1,269 +1,425 @@
-SRT provides a powerful set of statistical data on a socket.
-This data can be used to keep an eye on a socket's health,
-and track faulty behavior.
+# SRT Statistics
 
-Statistics are calculated independently on each side (receiver and sender),
-and are not exchanged between peers, unless explicitly stated.
+SRT provides a powerful set of statistical data on a socket. This data can be used to keep an eye on a socket's health and track faulty behavior.
 
-The following API functions can be used to retrieve statistics on an SRT socket.
-Refer to the documentation of the [API functions](API-functions.md) for usage instructions.
+Statistics are calculated independently on each side (receiver and sender) and are not exchanged between peers unless explicitly stated.
+
+TODO: unless explicitly stated ??? -> check this on msRTT, mbpsBandwidth
```

Comment:
The average RTT and Bandwidth measurements are being performed at the receiver side and sent back to sender through UMSG_ACK message. Mention this as exception and you don't need extra TODOs.